src/utils/index.ts

```diff
@@ -97,20 +105,23 @@ export const arrayBufferToBase64Url = (arrayBuffer: ArrayBuffer) => {
   return base64ToBase64url(base64);
 };
 
-export const fetchWithTimeout = async (resource:string, options?:RequestInit, timeout?:number) => {
-  
+export const fetchWithTimeout = async (
+  resource: string,
+  options?: RequestInit,
+  timeout?: number,
+) => {
   const controller = new AbortController();
   const id = setTimeout(() => controller.abort(), timeout ?? 10_000);
 
   const response = await fetch(resource, {
     ...options,
     headers: {
       ...options?.headers,
-      'Accept-Encoding': 'identity'
+      'Accept-Encoding': 'identity',
```

Comment:
100% sure we want to include this on every fetch?

Reply:
This was added by @dtfiedler in a previous PR

Reply:
it just ensures the request isn't compressed differently across gateways, resulting in different hashes for the same data